workspace/resource_notebook.go

```diff
@@ -389,7 +387,20 @@ func ResourceNotebook() *schema.Resource {
 			}
 			err = notebooksAPI.Create(createNotebook)
 			if err != nil {
-				return err
+				if isParentDoesntExistError(err) {
+					parent := filepath.ToSlash(filepath.Dir(path))
+					if parent != "/" {
```

Comment:
Will we ever have the else variant of this case? `/` is guaranteed to exist, so we will not enter the if block starting on 390.

Reply:
Right, but my point is the REST API will never return an error that / doesn't exist.

Reply:
I guess there isn't a problem with this per se, but it seems unnecessarily defensive, unless I'm missing something.

Reply:
ah, yes - misunderstood the case. let me change it...

Reply:
Fixed